pkg/apis/serving/v1beta1/component.go

```diff
@@ -80,6 +80,14 @@ type ComponentExtensionSpec struct {
 	// Maximum number of replicas for autoscaling.
 	// +optional
 	MaxReplicas int `json:"maxReplicas,omitempty"`
+	// ScaleTarget specifies scaling value of knative
```

Comment:
these two fields are now used by both knative and raw deployment mode, so we may want to reword here.